miniz_zip.c

```diff
@@ -649,6 +649,100 @@ static mz_bool mz_zip_reader_read_central_dir(mz_zip_archive *pZip, mz_uint flag
     cdir_size = MZ_READ_LE32(pBuf + MZ_ZIP_ECDH_CDIR_SIZE_OFS);
     cdir_ofs = MZ_READ_LE32(pBuf + MZ_ZIP_ECDH_CDIR_OFS_OFS);
 
+    /* 
+    ZIP files may contain more files, than "Total number of central directory records" in the EOCD can hold.
```

Comment:
When it may happen? What software generates such archives?

Reply:
I am not sure but I think it is possible for python to do it.